data_science/01.00-addition-laws.py

```diff
@@ -0,0 +1,416 @@
+# /// script
+# requires-python = ">=3.10"
+# dependencies = [
+#     "marimo",
+#     "matplotlib",
+#     "matplotlib-venn"
+# ]
+# ///
+
+import marimo
+
+__generated_with = "0.10.19"
+app = marimo.App()
+
+
+@app.cell(hide_code=True)
+def _(mo):
+    mo.md(
+        """
+        # 🎲 Addition Laws of Probability
+
+        Welcome to the world of combining probabilities! In this notebook, we'll learn how to:
+
+        - Add probabilities of simple events
+
+        - Work with mutually exclusive events
+
+        - Use the general addition rule
+
+        Let's make probability addition fun and visual! 🎯
+        """
+    )
+    return
+
+
+@app.cell(hide_code=True)
+def _(mo):
+    mo.md(
+        """
+        ## 🎯 Simple Events
+
+        Simple events are basic outcomes that don't overlap. Think of rolling a die:
+
+        - Rolling a 1 is a simple event
+
+        - Rolling a 2 is another simple event
+
+        - They can't happen at the same time!
+        """
+    )
+    return
```

Comment:
I think the concept of simple events is superfluous. I would have started directly with mutually exclusive events.